### PR TITLE
Update formula to include all Anka scripts

### PR DESCRIPTION
**Pull request by danielbayley**
## Pull Request 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other: (describe here) 
  
## Proposed Change

Issue Number: N/A

Also include scripts from [veertuinc/scripts](https://github.com/veertuinc/scripts)

## Breaking Change?

- [ ] Yes
- [x] No

## Checklist

Please validate that your PR fulfills the following requirements:
- [x] I have read the **CONTRIBUTING.md** documentation
- [x] I have branched from the master branch
- [x] I have checked that a similar PR does not exist or has been rejected in the past
- [x] PR changes are specific to a feature or bug
- [x] No style/format/cosmetic changes included
- [x] I have used existing style and naming patterns in my changes
- [x] Tests for the changes have been added/updated (if relevant)
- [x] Built/compiled and tested locally
- [x] Docs have been added/updated (if relevant)
- [x] I have squashed my changes into a single commit

### CODE REVIEW

File: Formula/anka-scripts.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-class AnkaGettingStartedScripts < Formula
+class AnkaScripts < Formula
```
Comment (NorseGaud) [inline Formula/anka-scripts.rb:1]: Please merge latest changes from master :)

**Review by NorseGaud — CHANGES_REQUESTED** [review]

File: Formula/anka-scripts.rb (hunk 1 above)
Comment (danielbayley) [reply Formula/anka-scripts.rb:1]: @NorseGaud Ok cool… Although now this formula also includes scripts from the other repo, so should it still share the `GettingStarted` name?

Comment (NorseGaud) [reply Formula/anka-scripts.rb:1]: Good point -- What do you think about separating these Formula so we end up with `anka-scripts` and `anka-getting-started-scripts` ?

Comment (danielbayley) [reply Formula/anka-scripts.rb:1]: @NorseGaud Could do… Do they need to be in separate repositories in the first place?

Comment (NorseGaud) [reply Formula/anka-scripts.rb:1]: Yep, the "scripts" are just helper scripts. Our getting started repo are helper scripts, but they're used by our CI and a bunch of user guides/docs so I want to keep it as lightweight as humanly possible 😅

**Review by NorseGaud — APPROVED** [review]

**Pull request merged**